src/Microsoft.ApplicationInsights.AspNetCore/ExceptionTrackingMiddleware.cs

```diff
@@ -1,42 +0,0 @@
-﻿namespace Microsoft.ApplicationInsights.AspNetCore
-{
-    using System;
-    using System.Threading.Tasks;
-    using Microsoft.ApplicationInsights.DataContracts;
-    using Microsoft.ApplicationInsights.Extensibility.Implementation;
-    using Microsoft.AspNetCore.Http;
-
-    /// <summary>
-    /// Sends telemetry about exceptions thrown by the application to the Microsoft Application Insights service.
-    /// </summary>
-    public sealed class ExceptionTrackingMiddleware
```

Comment:
Unhandled exception would still be reported by https://github.com/aspnet/Hosting/blob/b6da89f54cff11474f17486cdc55c2f21f2bbd6b/src/Microsoft.AspNetCore.Hosting/Internal/HostingApplication.cs#L79